sem_covid/services/pwdb_base_experiment.py

```diff
@@ -75,9 +77,8 @@ def data_validation(self, *args, **kwargs):
         pass
 
     def data_preparation(self, *args, **kwargs):
-        pwdb_json_dataset = json.loads(self.minio_adapter.get_object(config.PWDB_DATASET_LOCAL_FILENAME))
-        pwdb_dataframe = pd.DataFrame.from_records(pwdb_json_dataset)
-        pwdb_dataframe_columns = self.prepare_pwdb_data(pwdb_dataframe)
+        pwdb_dataset = Dataset.PWDB.fetch()
+        pwdb_dataframe_columns = self.prepare_pwdb_data(pwdb_dataset)
```

Comment:
we shall also start using the cleaning methods implemented by Stefan and abandon our older cleaning method